src/common-utils/src/data.ts

```diff
@@ -45,3 +49,23 @@ export function toArray<T>(item: T | T[]): T[] {
 
   return [item];
 }
+
+/**
+ * Check whether geojson linestring's 4th coordinate is 1) not timestamp 2) unix time stamp 3) real date time
+ * @param {array} timestamps array to be tested if its elements are timestamp
+ * @returns {Field | null} the type of timestamp: unix/datetime/invalid(not timestamp)
```

Comment:
Don't add types to JSDoc when typed via Typescript\
```suggestion
 * @param  timestamps array to be tested if its elements are timestamp
 * @returns the type of timestamp: unix/datetime/invalid(not timestamp)
```